src/components/Link/Link.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import type { StoryObj, Meta } from '@storybook/react';
 import React from 'react';
-import { Link } from './Link';
-import { SIZES, STATUSES, VARIANTS } from '../ClickableStyle';
+import { Link, type LinkProps } from './Link';
+import { SIZES } from '../ClickableStyle';
```

Comment:
should Sizes be removed or nah?

Reply:
oh good call. will tidy up what storybook shows for this one, and remove/hide that.